src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,91 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import React, { useMemo } from 'react';
+
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number | string) => void,
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pageAmount = useMemo(() => {
+    const pageCount = Math.ceil(total / perPage);
+
+    return getNumbers(1, pageCount);
+  }, [total, perPage]);
+
+  return (
+    <ul className="pagination">
+      <li className={classNames(
+        'page-item',
+        {
+          disabled: currentPage === 1,
+        },
+      )}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={() => {
+            if (currentPage > 1) {
+              onPageChange('prev');
+            }
```

Comment:
Maybe it will be better to create a handler function... Isn`t such logic too hard and complicated to place here?